awscli/examples/oam/delete-link.rst

```diff
@@ -0,0 +1,8 @@
+**To Delete a link**
+
+The following ``delete-link`` example deletes a link between a monitoring account sink and a source account. ::
+
+    aws oam delete-link \
+        --identifier arn:aws:oam:us-east-2:123456789111:link/a1b2c3d4-5678-90ab-cdef-example11111
+
```

Comment:
There is a missing output section. If this command produces no output please add the appropriate line.
https://aws.github.io/aws-cli/docs_styleguide.html#output